### ISSUE TITLE
Missing dependency information for advanced-achievements

### ISSUE THREAD

**Issue by Rsl1122** [PyvesB/AdvancedAchievements#361]
Hey!
Thanks for splitting the API to a separate package, and even including a repository for it. :)

----

There is an issue with the repo missing a pom for the main artifact, that causes maven to fail building.

In advanced-achievements-api pom.xml there is a decleration:
```
	<parent>
		<groupId>com.hm.achievement</groupId>
		<artifactId>advanced-achievements</artifactId>
		<version>5.7.4</version>
	</parent>
```

Since maven likes building full dependency graph before downloading anything, the build fails with an error:
> [ERROR] Failed to execute goal on project PlanPluginBridge: 
  Could not resolve dependencies for project com.djrapitops:PlanPluginBridge:jar:4.3.0-SNAPSHOT: 
  Failed to collect dependencies at com.hm.achievement:advanced-achievements-api:jar:1.0.0: 
  Failed to read artifact descriptor for com.hm.achievement:advanced-achievements-api:jar:1.0.0: 
  **Failure to find com.hm.achievement:advanced-achievements:pom:5.7.4** in 
  https://hub.spigotmc.org/nexus/content/repositories/snapshots/ was cached in the local repository, 
  resolution will not be reattempted until the update interval of spigot-repo has elapsed or updates are forced -> [Help 1]

The repositories mentioned are only there because maven attempts to find the pom of the parent file from every repository, but since it is unable to find it anywhere it errors out.

**Comment by PyvesB** [comment]
Hey, thanks for reporting this issue. It caught me off guard as I had all required artifacts installed in my local Maven repository whilst testing.

I have attempted a fix in #362. Could you please try to build again and see whether it fixes the encountered error?

**Comment by Rsl1122** [comment]
Required deletion of the local cached information about the dependency at `.m2/repository/com/hm/achievement`

Confirmed fixed.
